projects/example-app/tsconfig.app.json

```diff
@@ -1,4 +1,5 @@
 {
+  "extends": "../../tsconfig.json",
```

Comment:
Otherwise, we had problems to run the example app:

```
ERROR in Unable to write a reference to EffectsRootModule 
```

Followed the following doc https://hackmd.io/zhu28RUcRiaG0DZq5bLy1A?view